WORK-IN-PROGRESS

```diff
@@ -474,11 +476,75 @@ Modified: 2017-08-18
     I will not provide any user support for MinGW msys/1.0 in future.
     
     MinGW msys/2.0 might be supported as "experimental" in Exiv2 v0.26.2
-     
+    
+4.9 Building and linking XMPsdk with libexiv2
+
+    Exiv2 v0.26 (and many earlier editions) contain Adobe XMPsdk code.
+    The precise version of this code is unknown.
+    The code is stored in the xmpsdk/ directory which contains src/ and include/ directories.
+    The xmpsdk has it's own Makefile, CMakeLists.txt and msvc/xmpsdk/project files.
+    
+    Exiv2 v0.26 builds a static library libxmpsdk.a which is linked into libexiv2.a
+    
+    None of this is the change in Exiv2 v0.26.1
+    
+    Exiv2 v0.26.1 will support a new option to download and build "pure Adobe" XMPsdk.
```

Comment:
We could make explicit here that this option is for the moment only available with **the autotools** and that we will work in other ways to bring those dependencies in the future.